canonical/cloud-init
# d/cloud-init.templates: enable WSL datasource in packaging.

## rebase merge commits

## Proposed Commit Message
```
d/cloud-init.templates: enable WSL datasource in packaging.

Also update po translation files
```

## Additional Context
Now that WSL datasource has landed in tip of main, enable it in packaging by adding it to cloud-init.templates

## Test Steps
```
dhc -r -D noble ''
git commit -am 'test full package build'
build-package
sbuild --dist=noble --arch=amd64 --arch-all ../out/cloud-init_*.dsc
lxc launch ubuntu-daily:noble test-n
lxc file push cloud-init*_all.deb test-n/
lxc exec test-n -- apt -i /cloud-init*deb
lxc exec test-n -- dpkg-reconfigure cloud-init # confirm you can select  WSL from datasource search options
```

## Checklist


## Merge type
